terraform/environments/digital-prison-reporting/athena_federated_queries.tf

```diff
@@ -0,0 +1,32 @@
+locals {
+  # TODO Parameterise NOMIS IP
+  #  Dev or Test for now
```

Comment:
@harichintala1 I need to configure this for each environment. What do you think to having it set in application_variables.json?

Reply:
For this one we've agreed Hari will add static IPs when he gets a chance